modules/light-clients/07-tendermint/types/misbehaviour_handle.go

```diff
@@ -88,7 +89,7 @@ func (cs ClientState) CheckMisbehaviourAndUpdateState(
 		return nil, sdkerrors.Wrap(err, "verifying Header2 in Misbehaviour failed")
 	}
 
-	cs.FrozenHeight = tmMisbehaviour.GetHeight().(clienttypes.Height)
+	cs.FrozenHeight = clienttypes.NewHeight(0, 1)
```

Comment:
can we have a const `FrozenHeight`? This would be useful in tests

Reply:
Struct can't be a constant but I can make it a var inside tendermint package